src/inscriptions/types.ts

```diff
@@ -0,0 +1,20 @@
+import type { RequestOptions, RequestPayload } from '../types';
+
+export interface CreateInscriptionPayload extends RequestPayload {
+  contentType: string;
+  content: string;
+  payloadType: 'PLAIN_TEXT' | 'BASE_64';
+  inscriptionFee?: number;
+  feeAddress?: string;
```

Comment:
If we change `inscriptionFee` to `appFee` this should be changed to `appFeeAddress`

Reply:
Sounds good 👍 I changed all 3 variable names, they make a lot more sense now 😄 Thanks